app/src/androidTest/java/ch/epfl/balelecbud/EmergencyNumbersActivityTest.java

```diff
@@ -0,0 +1,36 @@
+package ch.epfl.balelecbud;
+
+import androidx.test.rule.ActivityTestRule;
+import android.view.View;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.assertion.ViewAssertions.matches;
+import static androidx.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static androidx.test.espresso.matcher.ViewMatchers.withId;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.IsNull.notNullValue;
+
+
+@RunWith(AndroidJUnit4.class)
+public class EmergencyNumbersActivityTest {
```

Comment:
The problem with this test is that you do not check the value displayed or for example that it updates